ifrs17/Import/ImportStorage.ipynb

```diff
@@ -273,15 +274,8 @@
                 "\n",
                 "\n        if(DefaultPartition != TargetPartition) {",
                 "\n            await querySource.Partition.SetAsync<PartitionByReportingNodeAndPeriod>(DefaultPartition);",
-                "\n            var defaultRawVariables = await querySource.Query<RawVariable>()",
-                "\n                                                       .Where(rv => rv.Partition == DefaultPartition)",
```

Comment:
why is this Where removed? Isn't it necessary for physical DB?

Reply:
yes, you are right, this needs to be there to support physical db